QPM/SymLinker.Linker/SymLinker.Linker.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Please enable nullable and at least C#8 (C#9 is perfectly fine) for `new()`. You use .net5, so might as well take advantage of it.